From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aditya Prayoga <aditya@kobol.io>
Date: Tue, 15 Sep 2020 20:04:22 +0700
Subject: arm64: dts: rockchip: helios64: add fan tachometer pin groups

Pin groups for the fan tachometer inputs GPIO4_C5/GPIO4_C7. Nothing
references them here.

Upstream: the same groups plus pwm-fan interrupts applied as
fc411363b823 (mmind v7.4-armsoc/dts64); drop on 7.4.

Split out of add-board-helios64.patch (Aditya Prayoga, rebased onto
mainline by rpardini for 6.3); the DTB content is unchanged.

Signed-off-by: Aditya Prayoga <aditya@kobol.io>
Assisted-by: LLM
Signed-off-by: Igor Velkov <iav@iav.lv>
---
 arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts b/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
@@ -494,6 +494,16 @@ &pinctrl {
 		};
 	};
 
+	fan {
+		fan1_sense: fan1-sense {
+			rockchip,pins = <4 RK_PC5 RK_FUNC_GPIO &pcfg_pull_none>;
+		};
+
+		fan2_sense: fan2-sense {
+			rockchip,pins = <4 RK_PC7 RK_FUNC_GPIO &pcfg_pull_none>;
+		};
+	};
+
 	gmac {
 		gphy_reset: gphy-reset {
 			rockchip,pins = <3 RK_PB7 RK_FUNC_GPIO &pcfg_output_low>;
-- 
Armbian

